index.html

```diff
@@ -1,26 +1,113 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html lang="en">
-  
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="device-width, initial-scale=1.0"> 
-    <link rel="stylesheet" href="styles.css"> 
-   </head>
-  
-  <body>
-    <header>Glazed Donuts</header> 
-    
-    <main> </main>
-    
-    
-    
-    
-    
-    
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372 --> 
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut02.jpg?v=1602793077492 --> 
-    
-    
-  </body>
-  
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fspiceracklogo.png?v=1603324422595" alt="">
```

Comment:
Before you turn in your final project, you'll want to add `alt` values to these and your other images.